src/main/index.js

```diff
@@ -121,7 +121,12 @@ export { downloadBlob } from './utils';
 /**
  * @typedef {Object} SourceLoadError
  * @property {string} url url of the ol.source that encountered at least one 'tileloaderror' or 'imageloaderror'.
- * /
+ */
+
+/**
+ * @typedef {true|'bottom-left'|'bottom-right'|'top-left'|'top-right'} WidgetPosition
```

Comment:
Strongly typing the possible widget positions is a good call! Although, I find it weird to put a boolean in the mix :/